Topic edits (#98)
HCL-TECH-SOFTWARE/digital-experience: docs/get_started/plan_deployment/traditional_deployment/roadmaps/rm_install_deployment/rm_advanced_cfg/rm_production_farm.md

# HCL Digital Experience roadmap: Portal farm

> A server farms offers a simple way to build and maintain a highly scalable, highly available server environment.

Reviewer: ```suggestion
A server farm offers a simple way to build and maintain a highly scalable, highly available server environment.
```

Author: Yes!

Gather information and software before you install HCL Digital Experience.

1.  Check requirements. See [Detailed system requirements](../../../../../system_requirements/index.md).

2.  Get required software. New and current users must [sign up or register at HCL Software](https://www.hcltechsw.com/wps/portal/about/welcome) and download their preferred HCL Digital Experience package.

## Installing prerequisite software

> You can use currently installed prerequisite software. However, verify that your current version is supported. If it is not, upgrade to a supported version. Otherwise, install a web server, database server, and user registry server. Typically the database and user registry servers are already installed and configured. However, you might have to complete specific configuration steps that are required to integrate the software with the portal server. For moreinformation, see "Installing and preparing the prerequisite software."

Reviewer: ```suggestion
You can use currently installed prerequisite software. However, verify that your current version is supported. If it is not, upgrade to a supported version. Otherwise, install a web server, database server, and user registry server. Typically, the database and user registry servers are already installed and configured. However, you might have to complete specific configuration steps that are required to integrate the software with the portal server. For more information, see "Installing and preparing the prerequisite software."
```

Author: Yes

<!-- The information in three-item list was presented in the preceding paragraph. -->

## Installing HCL Digital Experience

Installing HCL Digital Experience involves preparing your operating system, installing or upgrading the installation manager, and running the installation program. See [Installing the HCL Digital Experience software](../../../../../../deployment/install/traditional/installing_dx/index.md)

## Applying the latest cumulative fix

> You maintain the portal by deploying individual Fixes and Combined Cumulative Fixes (CFs), which are suggested according to your environment. See [HCL Digital Experience roadmap: Applying maintenance](../rm_cf.md)

Reviewer: ```suggestion
You maintain the portal by deploying individual Fixes and Combined Cumulative Fixes (CFs), which are suggested according to your environment. See [HCL Digital Experience roadmap: Applying maintenance](../rm_cf.md).
```

Author: Yes

    -   Click **Download Files** to run the steps remotely.
    -   Click **Run All Steps** to run the steps locally.

8.  If you changed the context root during the installation or configuration of HCL Digital Experience, then you must complete the optional next step from the Configuration wizard to update parameters with the new context path after you complete the Create a Deployment Manager configuration option. For more information about this configuration option and completing the next steps, see [Create a deployment manager](../../../../../../deployment/manage/config_cluster/cw_dmgr_profile.md).

9.  Log in to HCL Digital Experience to verify that the portal server works.

10. Select **Set Up a Stand-alone Server** \> **Enable Federated Security**.

    !!!note

>         If you set **Use Administrator IDs stored in your LDAP user registry** to yes, the and HCL Digital Experience user IDs and passwords are changed to the LDAP user ID and password. If you do not want to change both user IDs and passwords to match the LDAP user ID and password, set this value to no. After you configure your LDAP user registry, you can manually change the user IDs and passwords. See [Updating user ID and passwords](../../../../../../deployment/manage/security/people/authentication/updating_userid_pwd/index.md)

Reviewer: ```suggestion
        If you set **Use Administrator IDs stored in your LDAP user registry** to yes, the and HCL Digital Experience user IDs and passwords are changed to the LDAP user ID and password. If you do not want to change both user IDs and passwords to match the LDAP user ID and password, set this value to no. After you configure your LDAP user registry, you can manually change the user IDs and passwords. See [Updating user ID and passwords](../../../../../../deployment/manage/security/people/authentication/updating_userid_pwd/index.md).
```

Author: Yes

    -   Click **Download Files** to run the steps remotely.
    -   Click **Run All Steps** to run the steps locally.

14. Log in to HCL Portal to verify that the portal server works.

15. To shorten the site URL for search engine optimization, you can modify your context root and remove navigational state information from the URL by using the **Modify Site URLs for Search Engine Optimization** configuration option. See [Configuration Wizard instructions: Modify site URLs for search engine optimization](../../../../../../deployment/manage/siteurl_cfg/changing_siteurl/cw_shorten_url/index.md)

# Configuring the web server

> Move the web server plug-in from the WebSphere Application Server to the web server. See [Web servers](../../../../../../deployment/manage/config_webservers/index.md)

Reviewer: ```suggestion
Move the web server plug-in from the WebSphere Application Server to the web server. See [Web servers](../../../../../../deployment/manage/config_webservers/index.md).
```

Author: Yes

## Tuning the servers in your environment

> Tuning the servers enhances the performance of your portal environment. HCL Digital Experience is not tuned for a production environment after installation and deployment. Your database must be tuning for improved performance. You can organize your database now or soon after you finish your configuration. Tune and maintain your database on a regularly.

Reviewer: ```suggestion
Tuning the servers enhances the performance of your portal environment. HCL Digital Experience is not tuned for a production environment after installation and deployment. Your database must be tuned for improved performance. You can organize your database now or soon after you finish your configuration. Tune and maintain your database regularly.
```

Author: Yes

> 1.  Run the performance tuning tool to initially tune your servers. See [Portal server performance tuning tool](../../../../../../deployment/manage/tune_servers/wp_tune_tool.md)

Reviewer: ```suggestion
1.  Run the performance tuning tool to initially tune your servers. See [Portal server performance tuning tool](../../../../../../deployment/manage/tune_servers/wp_tune_tool.md).
```

Author: Yes

> 2.  Check the tuning guide for more instructions. Use the tuning guide for the previous product version until the tuning guide for the current release is unavailable. See [Performance tuning guide](https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0074411)

Author: Yes
